packages/zowe-explorer/src/uss/UssFSProvider.ts

```diff
@@ -472,7 +472,7 @@ export class UssFSProvider extends BaseProvider implements vscode.FileSystemProv
             // If file names match, build the copy suffix
             let dupCount = 1;
             const extension = path.extname(fileName);
-            const baseNameForFile = path.parse(fileName)?.name;
+            const baseNameForFile = path.parse(fileName).name;
```

Comment:
Interesting that we would add `?.` here.
I couldn't find which version of Node (since node12) would return a `null/undefined` value.

In any case, I would think we want to make sure that fileName is a string or surround the `baseNameForFile` assignment in a try/catch